docs/quickstart/ccd.md

```diff
@@ -0,0 +1,246 @@
+Your [Strm profile page][profile] should show: "Current subscription: Self hosted". Get in touch with your Strm
+representative if you're on a Free or Business subscription.
+
+Once you have this profile active, you can start.
+
+## Install the following tools
+
+* `make` (optional). The Helm repository contains handy `make` targets such as _install_, _upgrade_, _wipe_. None of
+  these are mandatory, one can also execute the commands in the [`Makefile`][makefile] by hand.
+* [`kubectl`](https://kubernetes.io/docs/tasks/tools/)
+* [`helm`](http://helm.sh). This Kubernetes package manager is used for installing (and upgrading) your Strm customer
+  dataplane.
+* [`telepresence`][telepresence]. This tool allows your development computer to become _part of_ the k8s cluster, so
+  that you can directly access k8s services.
+* [`k9s`](https://github.com/derailed/k9s) (optional). This _textual user interface_ offers a very convenient way to interact with
+  kubernetes clusters.
+* [`kubectx and kubens`](https://github.com/ahmetb/kubectx) (optional). Very useful tools to switch the default
+  kubernetes context and namespace.
+
+## Start the installation of the Strm customer data-plane
+
+1. download the credentials file `values.yaml` [here][values] into the `data-plane-helm-chart` directory.
+1. make sure you have active kubernetes credentials to a cluster: `kubectl get nodes` should show the nodes of your
+   cluster.
+1. Create a namespace `strmprivacy` and set that as default (`kubens strmprivacy`)
+1. `git clone https://github.com/strmprivacy/data-plane-helm-chart.git`. In the near future this will become part of a
+   Helm repository, and this command will become `helm repo add ...`
+1. `cd data-plane-helm-chart`
+
+The `values.yaml` file should be something like this
+
+    registry:
+      imagePullSecret: "ewogIC...."
+
+    license:
+      installationId: "f4cea...."
+      installationClientId: "ins-...."
+      installationClientSecret: "tii...."
+
+    kafka:
+      enabled: true
+
+    redis:
+      enabled: true
+
+    postgresql:
+      enabled: true
+
+## Create the helm release
+
+This installs _all_ the Strm components inside the `strmprivacy` namespace.
+
+`make install` should be all that is needed. `kubectl get pods --watch` or `k9s` provides nice feedback to see how the
+installation is progressing. We see that some supporting infrastructure like Redis, Postgresql and Kafka are also
+installed. The creation of these components can be disabled, in which case configuration to the actual components will
+have to be added to the Helm chart.
+
+### Create in a different namespace
+
+**this is broken, don't use it**
+
+You'll have to add `namespace=...` to every `make` command (or modify the `Makefile`)
+
+    make namespace namespace=mycustomnamespace
+    make install namespace=mycustomnamespace
+
+The install seems to work correctly but trying to send events seems to go into some black
+hole. *to be investigated*
```

Comment:
This text should really be removed. It looks very clumsy to first say something is broken, then explain how it works and then again mention that it doesn't work.
For now just add the text that installing in another namespace than `strmprivacy` is not yet fully operational.